### DISCUSSION TITLE
time_tolerance and sync_times

### DISCUSSION THREAD

**Discussion by hsheldon** [idaholab/moose#16285] [Q&A General] [answered]
Does time_tolerance apply to sync_times, or is it only used at the beginning and end of  the simulation? I'm having trouble with the solver not reaching sync_times exactly, then trying to take a tiny step to reach the next sync_time and failing to converge. Setting time_tolerance doesn't seem to make any difference.

**Comment by aeslaughter** [comment, marked as answer]
Yes, it it applied in the TimeStepper object. The "timestep_tolerence" in the Executioner is applied to the TimeStepper here: https://github.com/idaholab/moose/blob/ee00f87eabf9350ab0d2d644b9eea0163c938267/framework/src/timesteppers/TimeStepper.C#L130

**Reply by hsheldon** [reply]
Ah, I needed timestep_tolerance, not time_tolerance! Thanks. It works now.
@WilkAndy

**Reply by WilkAndy** [reply]
> Ah, I needed timestep_tolerance, not time_tolerance! Thanks. It works now.
> @WilkAndy

Ooops, sorry for setting you wrong @hsheldon

**Reply by aeslaughter** [reply]
Glad it works, I didn't even notice the difference between your question and what I found. 😄
